api/apitest/apitest.go

```diff
@@ -25,6 +25,11 @@ var SessionSecret = NewUsername()
 func MakeTextile(t *testing.T) (conf core.Config, shutdown func()) {
```

Comment:
Some small refactor to provide a custom config for running tests.
Needed in the integration test to tune up configs for it.